package.json

```diff
@@ -44,22 +44,42 @@
   "devDependencies": {
     "@commitlint/cli": "11.0.0",
     "@commitlint/config-conventional": "11.0.0",
-    "@typescript-eslint/eslint-plugin": "4.9.1",
-    "@typescript-eslint/parser": "4.9.1",
+    "@types/got": "9.6.11",
+    "@types/jquery": "3.5.5",
+    "@types/mocha": "8.2.0",
+    "@types/node": "14.14.12",
+    "@types/request-promise-native": "1.0.17",
+    "@types/semver": "7.3.4",
+    "@types/shimmer": "1.0.1",
+    "@types/sinon": "9.0.9",
+    "@types/superagent": "4.1.10",
+    "@types/webpack-env": "1.16.0",
+    "@typescript-eslint/eslint-plugin": "^4.10.0",
+    "@typescript-eslint/parser": "^4.10.0",
     "beautify-benchmark": "0.2.4",
     "benchmark": "2.1.4",
-    "eslint": "7.15.0",
+    "eslint": "^7.15.0",
     "eslint-config-airbnb-base": "14.2.1",
     "eslint-plugin-header": "3.1.0",
     "eslint-plugin-import": "2.22.1",
     "gh-pages": "3.1.0",
     "gts": "2.0.2",
```

Comment:
PR description says gts is gone

Reply:
Good catch. Some other extraneous deps removed too.